### ISSUE TITLE
Tab active state isn't seen in Windows high contrast mode

### ISSUE THREAD

**Issue by dougmacknz** [cultureamp/kaizen-design-system#1922]
TLDR: the active state can't be seen in windows high contrast mode, we need to use a border, or use a media query to adjust this.

## Audit info

Type:Expert observation
Principle: Not applicable
Guideline:Not applicable
Success Criteria: Not applicable
Level: Not applicable
Platform: Web
Severity:High

### What's the issue?
The indicator to show which tab has been selected can't be determined when viewed with Windows high contrast mode enabled, because it relies on the background-color property which is removed in this mode.

### Our recommendation
Make sure that all informative styles are visible in Windows high contrast mode.
Use an icon or border style instead of relying on removed CSS properties such as box-shadow and background colours.

## Screenshots

Not high contrast mode:
![image](https://user-images.githubusercontent.com/1811583/131786154-1d37c956-d3a5-4097-8e63-684a67ce55b7.png)

High contrast mode:
![summary-tabs-hc(2)](https://user-images.githubusercontent.com/1811583/131786089-f97ed8b1-2136-42d8-b82d-f676b807cb55.PNG)
